docs/_posts/0000-05-01-oo-inheritance.md

```diff
@@ -1,16 +1,14 @@
 ## Inheritance
 
 + What is it?
-    + Sharing functionality
-+ How do we achieve it?
     + Some notion of linking types together
-        + Classes and subclasses
-        + Proto-typical inheritance
-    + Composition (sharing functionality without inheritance)
+        + Super classes and subclasses
+        + Parents and children
```

Comment:
I’d like to leave prototypical inheritance in here as it’s slightly different to class-based inheritance.
```suggestion
        + Parents and children
        + Prototypical inheritance
```